docs/INSTALL.SSL.md

```diff
@@ -109,7 +115,7 @@ additional options.
 /usr/share/openvswitch/scripts/ovn-ctl \
     --ovn-controller-ssl-key="/etc/openvswitch/ovncontroller-privkey.pem"  \
     --ovn-controller-ssl-cert="/etc/openvswitch/ovncontroller-cert.pem"    \
-    --ovn-controller-ssl-bootstrap-ca-cert="/etc/openvswitch/ovnsb-ca.cert" \
+    --ovn-controller-ssl-bootstrap-ca-cert="/etc/openvswitch/ovnsb-cert.pem" \
     restart_controller
```

Comment:
What was your thought process to change the names here?

Reply:
I understand why you did it now. Makes sense.

Reply:
@shettyg All the certificates are named with `.pem` extension. So, to keep it consistent I changed the name. Furthermore, if a CA Certificate already exists in `/etc/openvswitch`, then `--ovn-controller-ssl-bootstrap-ca-cert` has the same effect as -`-ca-cert`. As we know ovn-controller talks to SB and if someone did copy the OVN SB CA Certificate from OVN Central, then it would have been named `ovnsb-cert.pem` (because `ovs-pki -b sign ovnsb` names the certificate as such).